mkalias/mkalias.c

```diff
@@ -20,7 +20,10 @@
 
 /*
   CHANGES
-
+ 
```

Comment:
Spurious spaces. I see them in my editor :)

![hidden-spaces](https://user-images.githubusercontent.com/451178/43547017-0a8f5304-95a8-11e8-98f5-266b560ac202.png)